lib/twilio-test-toolkit/call_in_progress.rb

```diff
@@ -1,56 +1,32 @@
 require "twilio-test-toolkit/call_scope"
-  
+
 module TwilioTestToolkit
   # Models a call
   class CallInProgress < CallScope
     # Initiate a call. Options:
     # * :method - specify the http method of the initial api call
     # * :call_sid - specify an optional fixed value to be passed as params[:CallSid]
     # * :is_machine - controls params[:AnsweredBy]
+    attr_reader :sid, :initial_path, :from_number, :to_number, :is_machine, :http_method
+
     def initialize(initial_path, from_number, to_number, options = {})
       # Save our variables for later
       @initial_path = initial_path
       @from_number = from_number
       @to_number = to_number
       @is_machine = options[:is_machine]
       @method = options[:method] || :post
+      @sid = options[:call_sid] ||= UUIDTools::UUID.random_create.to_s
 
-      # Generate an initial call SID if we don't have one
-      if (options[:call_sid].nil?)
-        @sid = UUIDTools::UUID.random_create.to_s
-      else
-        @sid = options[:call_sid]
+      if options[:method].nil?
```

Comment:
So - interesting...

The call_scope class uses the `options[:method]` so that's why we need both set. The options get passed into `request_for_twiml!`. The @method is really only a getter for those outside twiml to check the value. 

The other two that we talked about earlier weren't changing the value of the options hash so I removed them, but this one would set the default `:post` value into the options hash so I think we need it.

Reply:
Totally -

I was moreso thinking to do this:

```ruby
def initialize(...)
  options[:method] ||= :post # sets ot for the `request_for_twiml!`

  # ... 
  
  @method = options[:method]
```

Reply:
Gotcha - all set